contracts/optimistic-ethereum/libraries/codec/Lib_EIP155Tx.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.9.0;
```

Comment:
Pulled decoding logic into this file to make review easier. It's mostly the same as what was originally in OVM codec. We can confirm this with some careful review.